pyproject.toml

```diff
@@ -27,7 +27,7 @@ generateBuildrootSBOM = "cyclonedx_buildroot._internal.cli:run"  # for backwards
 [tool.poetry.dependencies]
 python = "<4.0,>=3.8"
 packageurl-python = "0.14.0"
-cyclonedx-python-lib = "8.0.0"
+cyclonedx-python-lib = "8.5.0"
```

Comment:
better get your version ranges widened via #61 instead of bumping min-versions without need. ;-)

Reply:
Needed this to test the features that are available in 8.5. I call this baby steps. One issue corrected at a time. No need to go to other versions yet.

Reply:
please use a concept of lowest required version and version ranges. see https://github.com/CycloneDX/cyclonedx-buildroot/pull/61

you are packaging this tool on pypi - so you have others that want to use the tool.
and they are maybe using it in an environment where they intend to "provide" a certain version of the runtime library - they dont want to use the exact version you used in your test env.

i don't see any of the features used, that were introduced after 8.0 - so your lowest supported version should be `^8.0`.

Reply:
The 8.5 release supports manufacturer, that is what I needed to test. Thus 8.5 is a valid starting point.

Reply:
i see .